definitions/private-law/json/CaseEvent/CaseEvent.json

```diff
@@ -278,6 +278,7 @@
     "PostConditionState": "*",
     "ShowSummary": "Y",
     "ShowEventNotes": "N",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/pre-populate-child-information",
```

Comment:
I somehow cant see the implementation of pre-populate-child-information endpoint, can we discuss on this pelase?

Reply:
As discussed backend changes are covered in https://tools.hmcts.net/jira/browse/PRL-5737